cms/static/js/i18n/ru/djangojs.js

```diff
@@ -244,7 +244,7 @@
     "Already a course team member": "\u0420\u0430\u043d\u0435\u0435 \u0447\u043b\u0435\u043d\u044b \u043a\u043e\u043c\u0430\u043d\u0434\u044b \u043a\u0443\u0440\u0441\u0430", 
     "Already a library team member": "\u0423\u0436\u0435 \u044f\u0432\u043b\u044f\u0435\u0442\u0441\u044f \u0447\u043b\u0435\u043d\u043e\u043c \u043a\u043e\u043c\u0430\u043d\u0434\u044b \u0432 \u0431\u0438\u0431\u043b\u0438\u043e\u0442\u0435\u043a\u0435", 
     "Already a member": "\u0423\u0436\u0435 \u0443\u0447\u0430\u0441\u0442\u0432\u0443\u0435\u0442", 
-    "Already have an account?": "\u0423\u0436\u0435 \u0435\u0441\u0442\u044c \u0443\u0447\u0451\u0442\u043d\u0430\u044f \u0437\u0430\u043f\u0438\u0441\u044c?", 
+    "Already have an account?": "Уже есть учетная запись?",
```

Comment:
Please regenerate djangojs.js. you can use this command:
```
../venvs/edxapp/bin/python ./manage.py lms compilejsi18n --setting=aws
```

Reply:
Done